src/map/skill.cpp

```diff
@@ -2441,12 +2441,11 @@ int skill_counter_additional_effect (struct block_list* src, struct block_list *
 		sc_start(src,src,SC_BLIND,2*skill_lv,skill_lv,skill_get_time2(skill_id,skill_lv));
 		break;
 	case HFLI_SBR44:	//[orn]
-	case HVAN_EXPLOSION:
 		if(src->type == BL_HOM){
-			TBL_HOM *hd = (TBL_HOM*)src;
-			hd->homunculus.intimacy = (skill_id == HFLI_SBR44) ? 200 : 100; // hom_intimacy_grade2intimacy(HOMGRADE_HATE_WITH_PASSION)
+			TBL_HOM *hd = BL_CAST(BL_HOM,src);
+			hd->homunculus.intimacy = 200; // hom_intimacy_grade2intimacy(HOMGRADE_HATE_WITH_PASSION)
```

Comment:
I have no idea about the skill, but judging from the description I think that both skills should set the intimacy to hom_intimacy_grade2intimacy(HOMGRADE_HATE_WITH_PASSION) instead of hardcoded 200/100.
Can we confirm that?

Reply:
Can't find any reliable source, apparently that 200 only appears in irowiki, but aegis probably just sets the intimacy to hate with passion

Reply:
@Lemongrass3110 I've been doing some research and found some old posts on iRO boards talking about S.B.R.44 reducing the intimacy by 100 * skill level (same as damage), but others say it just consumes all the intimacy, so no idea...

Reply:
Maybe @mrjnumber1 can help us here!